css/styles.css

```diff
@@ -41,11 +41,13 @@
 /* @group Columns */
 
 .ab-wrap {
-    margin: 20px 0 0 -30px;
+    margin: 0 0 0 -10px;
```

Comment:
By removing _one_ line of code in favor of _one_ other line, we actually do not change the overall amount of CSS.
This change reverted the `.ab-wrap` margin back to it's original state.